pyFAI/units.py

```diff
@@ -57,15 +57,15 @@
 
 CONST_hc = hc = scipy.constants.c * scipy.constants.h / scipy.constants.e * 1e7
 """Product of h the Planck constant, and c the speed of light in vacuum
-in Angstrom.KeV. It is approximativly equal to:
- 
-pyFAI reference 12.398419292004204
-scipy v1.3.1:   12.398419739640717
-scipy-1.4.0rc1: 12.398419843320026
+in Angstrom.KeV. It is approximatively equal to:
+
+- pyFAI reference: 12.398419292004204
+- scipy v1.3.1:   12.398419739640717
+- scipy-1.4.0rc1: 12.398419843320026
```

Comment:
I find this section misleading. What is `pyFAI reference` ? What value can the user expect ?

Reply:
This constant used to be hard-coded. Now it relies on scipy which is probably better ... but when scipy changes its value, some tests needs to be adapted. It is good to keep some track of those constants when some tests are randomly failing.